### PR TITLE
Journal Week 3

### PR DESCRIPTION
**Pull request by ericksonk**

### CODE REVIEW

File: tutorial-week-3.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
+## How to Write a CSS Rule
+
+All CSS rules contain two parts, a ** selector ** and a ** declaration **. The ** selector ** indicates which HTML element you want the rule to apply to. The ** declaration ** indicates _how_ the selected element should be styles. It consists of 2 parts, a ** property ** and a ** value ** which are separated by a colon. The ** property ** indicates the aspect of the elements you want to change and the ** value ** specifies the settings you want to use for the given property.
```
Comment (Ricecoder) [inline tutorial-week-3.md]: Remove spaces after ** in order to bold the text.

**Comment by Ricecoder** [comment]
Good start! :+1:

**Comment by cmauban** [comment]
Nice intro on CSS rules :+1: it is very time consuming and a lot of guess and check. Since we guess and check a lot code comments help!

**Comment by nickycadavillo** [comment]
:+1: Looks great!  I agree with your section on the time consuming part of CSS!

**Pull request merged**